doc/Type/Sequence.pod6

```diff
@@ -7,8 +7,13 @@
     role Sequence does PositionalBindFailover { }
 
 This role implements a series of methods for converting sequences and
-accessing their elements. These methods have in common that they all
-access the underlying iterator if the sequence has not been
+accessing their elements. They should work on all sequences, regardless
+of how parallel they are: serial L<Seq|/type/Seq>, ordered parallel
```

Comment:
"how parallel" does not sound too good. To be precise, they are potentially concurrent, not really parallel.

Reply:
`RaceSeq` and `HyperSeq` are certainly about parallel processing, rather than concurrency.

Reply:
Probably "whether the operations are parallelized" is perhaps better than "how parallel", though.

Reply:
How about:

    They should work on all sequences, regardless of what (if any) parallelism they support:

Reply:
Less is more. The listed classes have a serial/parallel adjective already, so there's no need to mention parallelism in the introduction of the list.

Reply:
@jnthn can you please clarify? From my POV, concurrency provides an abstract layer to use parallelism *if the capacity is available*. It does not care if it's not.